P/pprof/build_tarballs.jl

```diff
@@ -6,8 +6,8 @@ name = "pprof"
 # so we use Go's pseudo version
 # `TZ=UTC git show --quiet --date='format-local:%Y%m%d%H%M%S' --format="%cd" $hash`
 
-hash = "f9b734f9ee64d0f5b63636a45cc77ed2744997ab"
-timestamp = "20191205061153"
+hash = "20978b51388db0648809a2c5cc88b494c7945ec1"
+timestamp = "20201109224723"
 version = Base.VersionNumber("0.0.0-$timestamp")
```

Comment:
We're going to decouple the version number of the JLL package from the upstream version number.  Also, using the prerelease part of the version number isn't really great for the package manager.  I suggest using
```suggestion
version = v"0.1.0"
```
or something like that

Reply:
I love it! 😁 I'm supportive if you are!

I'll commit your change, now. 👍